### ISSUE TITLE
OS/X free build (from website) cannot find libraries

### ISSUE THREAD

**Issue by WetDesertRock** [johanmattssonm/birdfont#4]
When launching BirdFont from the .app in the finder it cannot find the libraries it needs to load. It is worth noting that I am using the free version downloaded from the website, and haven't compiled this myself.

Here is a list of libraries it will try to load:
```
$ otool -L BirdFont.app/Contents/MacOS/BirdFont 
BirdFont.app/Contents/MacOS/BirdFont:
	/opt/birdfont/lib/libgee.2.dylib (compatibility version 3.0.0, current version 3.0.0)
	/opt/birdfont/lib/libgobject-2.0.0.dylib (compatibility version 4001.0.0, current version 4001.0.0)
	/opt/birdfont/lib/libglib-2.0.0.dylib (compatibility version 4001.0.0, current version 4001.0.0)
	/opt/birdfont/lib/libintl.8.dylib (compatibility version 10.0.0, current version 10.2.0)
	/opt/birdfont/lib/libfreetype.6.dylib (compatibility version 17.0.0, current version 17.2.0)
	/opt/birdfont/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.8)
	/opt/birdfont/lib/libgio-2.0.0.dylib (compatibility version 4001.0.0, current version 4001.0.0)
	/opt/birdfont/lib/libcairo.2.dylib (compatibility version 11203.0.0, current version 11203.14.0)
	/opt/birdfont/lib/libbirdfont.dylib (compatibility version 1.0.0, current version 1.0.0)
	/System/Library/Frameworks/Cocoa.framework/Versions/A/Cocoa (compatibility version 1.0.0, current version 15.0.0)
	/System/Library/Frameworks/WebKit.framework/Versions/A/WebKit (compatibility version 1.0.0, current version 531.9.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 123.0.0)
	/usr/lib/libobjc.A.dylib (compatibility version 1.0.0, current version 227.0.0)
	/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 550.0.0)
	/System/Library/Frameworks/AppKit.framework/Versions/C/AppKit (compatibility version 45.0.0, current version 1038.0.0)
	/System/Library/Frameworks/ApplicationServices.framework/Versions/A/ApplicationServices (compatibility version 1.0.0, current version 38.0.0)
	/System/Library/Frameworks/Foundation.framework/Versions/C/Foundation (compatibility version 300.0.0, current version 751.0.0)
```

As you see it is trying to load them from the directory /opt/birdfont/lib/ while they are really located in BirdFont.app/Contents/MacOS/

I have a python script modified from a previously written one that I used before to solve this issue. The fix is most likely something in your build system, this is what I found to hackishly fix the problem:
```python
import os
DRY = True
OUTPUT = "BirdFont.app/Contents/MacOS/BirdFont"

print("Preparing dylibs")
libs = os.popen("otool -L %s"%OUTPUT).read().split('\n')
for l in libs:
  l = l.strip()
  if l.startswith('/opt/birdfont/lib/'):
    l = l.split(' ')[0]
    cmd = 'install_name_tool -change {oldname} @executable_path/{libname} {target}'.format(
      oldname=l,
      libname=os.path.basename(l),
      target=OUTPUT
    )
    if not DRY:
      os.system(cmd)
    
    print(cmd)

#also need a manual:
# $ install_name_tool -change @executable_path/libbirdfont.dylib @executable_path/libbirdfont.36.0.dylib BirdFont.app/Contents/MacOS/BirdFont
```

**Comment by johanmattssonm** [comment]
Thank you. I will create a new release.

**Comment by johanmattssonm** [comment]
A new release is availabe. I believe this error is fixed in 2.3.3.
